python/lib/core/dmod/core/meta_data.py

```diff
@@ -94,9 +97,8 @@ class DataFormat(PydanticEnum):
     """
     AORC_CSV = (0,
```

Comment:
So I deliberately changed this here because I noticed the data I was seeing was different than the previous fields.  And the data I had were CSVs I'd regridded myself using @jduckerOWP utility.  

@aaraney, you're talking about the official AORC format.  For better or worse, that's not _exactly_ the same data format we see (or at least have seen) in the CSV regriddings, and I am fairly sure this was intentional.  In particular, (aside: ug, [this old chestnut](https://github.com/NOAA-OWP/ngen/issues/509#issuecomment-1503385622) again), the CSVs include `RAINRATE` instead of `APCP_surface`.

There probably should be something formal and official outside of DMOD (and ngen) defining NextGen regridded CSV and NetCDF data formats, though I don't think that there is right now (@hellkite500?).

Reply:
I found a working link to some documentation (https://www.weather.gov/media/owp/operations/aorc_v1_1_methods.pdf).

Reply:
After doing a little more digging, I am thinking these names are coming from the NWM AORC 1.1 dataset. I pulled a netcdf forcing file from the [NWM 3.0 retrospective dataset](https://noaa-nwm-retrospective-3-0-pds.s3.amazonaws.com/index.html#CONUS/netcdf/FORCING/1979/) and dumped and sorted the variable names:

```
char crs ;
double x(x) ;
double y(y) ;
float RAINRATE(time, y, x) ;
int LQFRAC(time, y, x) ;
int LWDOWN(time, y, x) ;
int PSFC(time, y, x) ;
int Q2D(time, y, x) ;
int reference_time(reference_time) ;
int SWDOWN(time, y, x) ;
int T2D(time, y, x) ;
int time(time) ;
int U2D(time, y, x) ;
int V2D(time, y, x) ;
```

These line up perfectly with the (WRF-Hydro forcing inputs)[https://ral.ucar.edu/sites/default/files/public/projects/wrf-hydro/technical-description-user-guide/wrf-hydrov5.2technicaldescription.pdf] (table 5.4):

```
SWDOWN Incoming shortwave radiation W/m2
LWDOWN Incoming longwave radiation W/m2
Q2D Specific humidity kg/kg
T2D Air temperature K
PSFC Surface pressure Pa
U2D Near surface wind in the u-component m/s
V2D Near surface wind in the v-component m/s
RAINRATE Precipitation rate mm/s or kg/m2/s
```

Reply:
What i'm getting at is, what we are calling `AORC` forcing are spatially aggregated `AORC` forcing that closely follow the `WRF-Hydro` forcing naming convention.

Reply:
When really, this code is just making sure they follow the `WRF-Hydro` forcing naming conventions.

Reply:
To be clear, I don't think this is on us. There seems to be overloading of the term _`AORC`_ forcing in several places amongst the ORG.

Reply:
I opened #613 so that this conversation has a place to continue.  I haven't prioritized it highly, but I wouldn't argue if you felt like it needed to be closer to the top of our list, so feel free to adjust the priority or move it up the board.